### ISSUE TITLE
General and accessibility issues with "Opportunities to Volunteer" page

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/FreeCENMigration#883]
As my attention was drawn to the page by another issue, I noticed some things about the page and in particular about the "Proceed for Registration" button: there are 3 issues:

1. The background colour is related to the **old colour scheme** (white on a brownish-orange and hard-coded in Refinery, partly because of the hover state)
2. The **colour contrast ratio is a mere 2.4:1**
3. There is **no visible focus on tabbing** (because the hard-coded style does not include a focus state)
 
As the page is publicly visible, colour combos should have a contrast ratio of at least 3.1:1 for large text (4.5:1 for ordinary text).

Which of the new colours should be used here? The default CEN link/button colours seem a logical choice.

I also noticed some typos, some clunky phrasing, and an overly-complex page layout with an illogical source order — I'm pretty sure the latter could be improved (at least) whilst still allowing the ad-block to remain fixed (sticky).

**Comment by Captainkirkdawson** [comment]
The colour scheme should be the standard CEN one for CEN.

It was looking to fix that when I found the issue with inability to access the html through refinery😒

Happy for you to fully clean up. IMHO the ad bloc is not really required if it is causing a pain

**Comment by PatReynolds** [comment]
@AlOneill is this still the case - if so can you move to backlog.  If not, please close.

**Comment by AlOneill** [comment]
Issues have been fixed, so closing.
